### ISSUE TITLE
Homebrew installation

### ISSUE THREAD

**Issue by stevegrunwell** [stevegrunwell/asimov#2]
Since Asimov is a macOS-specific utility for developers, it would be great if it could easily be installed via [Homebrew](https://brew.sh/).

I've worked through [the Homebrew Formula Cookbook](https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md) and put together a formula that's _syntactically_ correct:

```rb
class Asimov < Formula
  desc "Automatically exclude development dependencies from Time Machine backups"
  homepage "https://github.com/stevegrunwell/asimov"
  url "https://github.com/stevegrunwell/asimov/archive/v0.1.0.tar.gz"
  sha256 "0fc0bf66f16ecb94ab711f92ac9107f301f50ec98a28e141d7b14f006d1f13b8"
  head "https://github.com/stevegrunwell/asimov.git", :branch => "develop"

  def install
    bin.install "asimov"
  end

  test do
    system "#{bin}/asimov"
  end
end
```

Unfortunately (for me), Homebrew seems to put some requirements on new formulas:

```sh
$ brew audit --new-formula asimov
asimov:
  * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
  * GitHub repository too new (<30 days old)
Error: 2 problems in 1 formula
```

I'll hold onto the formula for now and hope a blog post helps get some traction on Asimov. In the meantime, uh...stars, watchers, and forks are welcome! 😄

**Comment by rowanbeentje** [comment]
As @quarterto points out, this is worth committing as-is so that it'd be possible to `brew tap stevegrunwell/asimov && brew install asimov`

**Comment by apaleslimghost** [comment]
more info: https://docs.brew.sh/Taps. note especially that the formula itself would have to live in a `homebrew-asimov` repo

**Comment by stevegrunwell** [comment]
Holy cow, how did I miss that? Thank you both for bringing this up — nothing beats the convenience of installing tools like this via Homebrew.

**Comment by jmarceli** [comment]
Hi is it possible to install it now will Homebrew somehow?
I've tried `brew tap stevegrunwell/asimov https://github.com/stevegrunwell/asimov` but after that, I was unable to execute `brew install stevegrunwell/asimov/asimov`.

**Comment by rowanbeentje** [comment]
@jmarceli not yet - the project needs some metadata to be set up. This ticket will track that.

(In the meantime, https://github.com/stevegrunwell/asimov#installation is the only way)

**Comment by muuvmuuv** [comment]
When this is installable via homebrew we should add a guide on how to remove it from the machine first. For those who have installed it already.

**Comment by muuvmuuv** [comment]
@all we should tell our friends to fork and star this repo. I really would like to have this as a brew package!

**Comment by rowanbeentje** [comment]
One more watcher and we're there!

**Comment by tomtastic** [comment]
I think we're there now

**Comment by muuvmuuv** [comment]
How is the status here?

**Comment by 0xdevalias** [comment]
Just came across project this and was wondering if there is any updated ETA for this issue?

**Comment by stevegrunwell** [comment]
For those following along at home, I _may_ have [submitted a PR](https://github.com/Homebrew/homebrew-core/pull/55921) while my daughter was having some "quiet time" today 😉

**Comment by stevegrunwell** [comment]
I'm proud to announce that as of today, Asimov is available via Homebrew!

<img width="563" alt="Asimov appearing under the New Formulae heading when running `brew update`" src="https://user-images.githubusercontent.com/233836/84819651-5b75b100-afe6-11ea-8697-868e4d738641.png">

Right now, the version available through Homebrew is 0.2.0, which is a bit dated; will work on getting a newer release prepared soon. If you want the latest version, you can install via:

```sh
$ brew install asimov --head
```
